sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -232,6 +236,7 @@ Licensed under the MIT License.
                 <include>io.dropwizard.metrics:metrics-core:[4.1.0]</include> <!-- {x-include-update;cosmos_io.dropwizard.metrics:metrics-core;external_dependency} -->
                 <include>io.micrometer:micrometer-core:[1.5.6]</include> <!-- {x-include-update;io.micrometer:micrometer-core;external_dependency} -->
                 <include>org.slf4j:slf4j-api:[1.7.30]</include> <!-- {x-include-update;org.slf4j:slf4j-api;external_dependency} -->
+                <include>org.hdrhistogram:HdrHistogram</include> <!-- {x-include-update;org.hdrhistogram:HdrHistogram;external_dependency} -->
```

Comment:
I believe the cosmos_ prefix is necessary in this tag as well.

Reply:
Good catch, yes it is required.

Reply:
Done